src/App.tsx

```diff
@@ -15,36 +16,90 @@
   'Garlic',
 ];
 
+const SORT_FILD_ALPHABETICCALY = 'alphabeticcaly';
+const SORT_FILD_LENGTH = 'length';
+
+function getPreparedGoods(goods: string[], sortField: string, rever: boolean) {
+  let newVisibleGoods = [...goods];
+
+  if (sortField === SORT_FILD_ALPHABETICCALY) {
+    newVisibleGoods.sort();
+  }
+
+  if (sortField === SORT_FILD_LENGTH) {
+    newVisibleGoods.sort((good1, good2) => good1.length - good2.length);
+  }
+
+  if (rever) {
+    return newVisibleGoods.reverse();
+  }
+
+  if (sortField === '') {
+    newVisibleGoods = [...goods];
+  }
+
+  return newVisibleGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
+  const [rever, setRever] = useState(false);
```

Comment:
The state variable `rever` is likely a typo. It should be `reverse` to match its intended functionality.